invenio_app_ils/records/loaders/schemas/json/internal_location.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2018 CERN.
+#
+# invenio-app-ils is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Internal Location schema for marshmallow loader."""
+
+
+from marshmallow import Schema, fields
+
+
+class InternalLocationSchemaV1(Schema):
+    """Internal Location schema."""
+
+    location_pid = fields.Str(required=True)
```

Comment:
About this, we have to fix `invenio-records-rest`. See: https://github.com/inveniosoftware/invenio-records-rest/issues/232